tests/test_info.py

```diff
@@ -186,3 +186,20 @@ def test_plots(self, fx_ti_2020_02_13, method):
         assert isinstance(fig, matplotlib.figure.Figure)
         for ax in axes:
             assert isinstance(ax, matplotlib.axes.Axes)
+
+    def test_draw_pango_lineage_subtree(self, fx_ti_2020_02_13):
+        ti = fx_ti_2020_02_13
+        svg = ti.draw_pango_lineage_subtree("A")
+        svg2 = ti.draw_subtree(tracked_pango=["A"])
+        assert svg == svg2
+        assert svg.startswith("<svg")
+        for u in ti.pango_lineage_samples['A']:
+            assert f"node n{u}" in svg
+
+    def test_draw_subtree(self, fx_ti_2020_02_13):
```

Comment:
These don't do anything? Better to delete if they don't do any testing, will give a false sense of coverage otherwise.

Reply:
Do they not do anything? I intended them to work? Have I missed something?

Reply:
Oops, sorry: I must have missed a commit. Hang on.

Reply:
Apologies: my editor didn't save some changes before commit. Fixed now. These tests should work.